Dockerfile

```diff
@@ -1,8 +1,26 @@
+#
+# Build certain packages from source (bleeding edge)
+#
+
+# Build Helmfile
+FROM golang:1.10 as helmfile
+ARG HELMFILE_VERSION=master
+ARG HELMFILE_REF=af1914c57530eb0a33ea4710bf7e0659a6eb2469
+WORKDIR /go/src/github.com/roboll/helmfile 
+RUN git clone -b ${HELMFILE_VERSION} https://github.com/roboll/helmfile.git .
+RUN git checkout ${HELMFILE_REF}
```

Comment:
Why we need this and
```
export HELMFILE_VERSION ?= 0.19.0
 ## Install helmfile to easily deploy collections of helm charts
 helmfile:
 	$(call github_download_binary_release,$(HELMFILE_VENDOR),v$(HELMFILE_VERSION),$@_$(OS)_$(ARCH))
```

Reply:
So I'm torn. The `packages` repo was designed to facilitate installation on not just linux but also mac (e.g. local dev harness). This is needed for local installations.